platform/msm_shared/display_menu.c

```diff
@@ -345,7 +345,7 @@ void display_bootverify_option_menu_renew(struct select_msg_info *msg_info)
 		FBCON_COMMON_MSG, big_factor);
 	display_fbcon_menu_message("Press volume keys to navigate, and "\
 		"press power key to select\n\n", FBCON_COMMON_MSG, common_factor);
-
+	display_fbcon_menu_message("Welcome to HICODE002 BOOTLOADER!\n", FBCON_RED_MSG, common_factor);
```

Comment:
Remember to drop customised parts. You can split these into another commit, which would be easier to manage.

Reply:
> Refe

Deleted